docs/notebooks/Berger_POPC_25ns_buildH_calcOP.ipynb

```diff
@@ -0,0 +1,1394 @@
+      "Dealing with frame 2 at 20.0 ps.\n",
+      "Dealing with frame 3 at 30.0 ps.\n",
+      "Dealing with frame 4 at 40.0 ps.\n",
+      "^C\n",
+      "Traceback (most recent call last):\n",
+      "  File \"/home/fuchs/software/miniconda3/envs/buildh/bin/buildH\", line 33, in <module>\n",
+      "    sys.exit(load_entry_point('buildh', 'console_scripts', 'buildH')())\n",
+      "  File \"/home/fuchs/papers/buildH_Amelie/buildH/buildh/cli.py\", line 234, in main\n",
+      "    core.fast_build_all_Hs_calc_OP(universe_woH, begin, end, dic_OP, dic_lipid, dic_Cname2Hnames)\n",
+      "  File \"/home/fuchs/papers/buildH_Amelie/buildH/buildh/core.py\", line 528, in fast_build_all_Hs_calc_OP\n",
+      "    Hs_coor = buildHs_on_1C(Cname_position, typeofH2build, helper1_coor, helper2_coor, helper3_coor)\n",
+      "  File \"/home/fuchs/papers/buildH_Amelie/buildH/buildh/core.py\", line 61, in buildHs_on_1C\n",
+      "    H1_coor, H2_coor, H3_coor = hydrogens.get_CH3(atom,\n",
+      "  File \"/home/fuchs/papers/buildH_Amelie/buildH/buildh/hydrogens.py\", line 117, in get_CH3\n",
+      "    unit_vect_He = geo.apply_rotation(v2, rotation_axis, theta)\n",
+      "  File \"/home/fuchs/papers/buildH_Amelie/buildH/buildh/geometry.py\", line 135, in apply_rotation\n",
+      "    return normalize(vec_rotated)\n",
+      "  File \"/home/fuchs/papers/buildH_Amelie/buildH/buildh/geometry.py\", line 22, in normalize\n",
+      "    return vec / norm(vec)\n",
+      "  File \"/home/fuchs/papers/buildH_Amelie/buildH/buildh/geometry.py\", line 37, in norm\n",
+      "    return np.sqrt((vec**2).sum())\n",
+      "  File \"/home/fuchs/software/miniconda3/envs/buildh/lib/python3.8/site-packages/numpy/core/_methods.py\", line 47, in _sum\n",
+      "    return umr_sum(a, axis, dtype, out, keepdims, initial, where)\n",
+      "KeyboardInterrupt\n"
+     ]
+    }
+   ],
+   "source": [
+    "!buildH endCONF.gro -l Berger_PLA -d order_parameter_definitions_MODEL_Berger_POPC.def \\\n",
+    "    -x popc0-25ns.trr -o OP_POPC_Berger_25ns.out"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "On the above cell, I launched buildH to show you how it works. For each frame handled by buildH, a message `Dealing with frame XXX at YYY ps.` is printed to the output. Because it takes some time and because it should have printed 2500 `Dealing with frame...`, I just pressed Ctrl-C to get this notebook not too much polluted by buildH output. This explains why there is a Python `KeyboardInterrupt`.\n",
+    "\n",
+    "Note that, you can also launch buildH in a regular shell that way:\n",
+    "\n",
+    "```\n",
+    "$ buildH endCONF.gro -l Berger_PLA -d order_parameter_definitions_MODEL_Berger_POPC.def \\\n",
+    "    -x popc0-25ns.trr -o OP_POPC_Berger_25ns.out\n",
+    "```\n",
+    "\n",
+    "Or that way:\n",
+    "\n",
+    "```\n",
+    "$ buildH endCONF.gro -l Berger_PLA -d order_parameter_definitions_MODEL_Berger_POPC.def \\\n",
```

Comment:
Pourquoi pas rajouter un flag -v ? Plutôt que de rediriger vers /dev/null ?

Reply:
We thought about that, we'll probably use some kind of *counter* (on the command line) telling the user how much of the calculation is left.